feast-ui/rockcraft.yaml

```
@@ -0,0 +1,60 @@
+name: feast-ui
+version: "0.49.0"
+summary: Feast UI Rock for Feature Store Visualization
+description: |
+  This Rock provides a containerized environment for running the Feast UI,
```

nitpick: Lowercase for rocks
```suggestion
  This rock provides a containerized environment for running the Feast UI,
```